src/components/AppContent/AppContent.vue

```diff
@@ -24,14 +24,37 @@
 
 <template>
 	<main id="app-content-vue" class="app-content no-snapper">
-		<!-- @slot Provide content to the app content -->
-		<slot />
+		<div id="app-content-wrapper" :style="`height: ${contentHeight}px;`">
+			<!-- @slot Provide content to the app content -->
```

Comment:
The documentation is misleading. It make it seems  like you are using this default slot to put the main content in here.